### PR TITLE
Clear Image Selection

### PR DESCRIPTION
**Pull request by cwmanning**
Adds the ability to clear existing image selections. Addresses #16

screen shot 2015-10-14 at 4 42 36 pm

### CODE REVIEW

File: src/components/__tests__/picker-test.jsx [hunk 1]
```diff
@@ -72,6 +72,24 @@ describe("Picker", function() {
     })
   })
 
+  describe("when a picker's clear selection button is clicked", function() {
+    let onExit      = sinon.spy()
+    let onChange    = sinon.spy()
```
Comment (nhunzaker) [inline src/components/__tests__/picker-test.jsx]: Do these need to be spies, or could they be NOOPs like `() => {}`?

File: src/mixins/record.js [hunk 2]
```diff
@@ -18,9 +18,11 @@ module.exports = {
   },
 
   fetchIf(slug) {
-    if (slug != undefined) {
+    if (slug) {
```
Comment (nhunzaker) [inline src/mixins/record.js]: What if the slug is 0?

File: src/components/__tests__/picker-test.jsx (hunk 1 above)
Comment (cwmanning) [inline src/components/__tests__/picker-test.jsx]: They were spies all over, but most didn't need to be. Fixed now.

File: src/mixins/record.js (hunk 2 above)
Comment (cwmanning) [inline src/mixins/record.js]: Good catch. I don't know why I changed that line at all, it's been restored.

File: src/components/__tests__/picker-test.jsx (hunk 1 above)
Comment (nhunzaker) [inline src/components/__tests__/picker-test.jsx]: Haha, good call. Sorry to begin that convention.

**Comment by nhunzaker** [comment]
Nice, :+1:

**Comment by mzlock** [comment]
:+1:

**Pull request closed without merging**